Ktisis/Structs/Bones/BoneCategories.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Ktisis.Structs.Bones {
+	public class BoneCategories {
+
+		public static readonly List<Category> Categories = new() {
```

Comment:
Suggest moving this to the `Category` class itself.
Might also be worth converting this back into a dictionary (with parameters `<string, Category>`) for easier lookup.